deck.py

```diff
@@ -5,7 +5,21 @@
 class Deck:
     suits = ["Clubs", "Hearts", "Spades", "Diamonds"]
 
-    values = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"]
+    values = [
```

Comment:
You are going to get a merge conflict here.

Reply:
I can manually resolve that though, right?

Reply:
Yes. When you merge this branch up with `master`, you'll get conflicts. My money says `PyCharm` has a way to help you deal with it. If it doesn't, you can handle it in the terminal. If you have problems figuring out how to interpret the merge conflicts, let me know. They can be confusing.

Reply:
I thought Github would flag the conflict and help me resolve it

Reply:
GH did flag it. IF you click on the conversation tab and scroll to the bottom it'll show up. GH might have a way to help, I've never used it before. We can jump on a video call if you need help.

Reply:
Nah, I think I got it. I have used the GH editor before.